bot/utils/exceptions.py

```diff
@@ -1,4 +1,14 @@
+
 class UserNotPlayingError(Exception):
     """Raised when users try to use game commands when they are not playing."""
 
     pass
+
+
+class ExternalAPIError(Exception):
+    """Raised when an external API(eg. Wikipedia) returns an error."""
+
+    def __init__(self, api: str):
+        super().__init__()
+        self.api = api
+    pass
```

Comment:
Good addition, can we add `status_code` and an optional argument that contains the error message received by the API too?

Reply:
Good point, right now it's too specific to the Wikipedia API
(that returns 200 even if it sends back an error :D )